### ISSUE TITLE
Validation errors

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#473]
# Semantics

Visibility: 3
Evaluation: Does not support
Success Criteria: 4.1.1: Parsing
Needs design: 

### Description
Each page and state should be run through a [validator](https://validator.w3.org/nu/) to ensure the markup structure is correct. While most/all pages have validation issues, they tend to be related to invalid markup nesting (e.g a <div> in a <span>) or invalid attributes (e.g. “row” attribute on an <input>).

![]()
